### PR TITLE
Product Listing Page made

### PR DESCRIPTION
**Pull request by Asypher00**
Check products.css for code
Screenshot (22)

### CODE REVIEW

File: products.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
+body {
+    background-color: white;
+    color: black;
```
Comment (rittik-ghosh) [inline products.css:7]: You can use variables

File: products.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    font-family: "comic sans ms";
+
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
```
Comment (rittik-ghosh) [inline products.css:15]: You can create and use utility classes to make the code clean

File: products.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+.nav-container {position:fixed;
+    left : 1vw;
+   z-index: 1;
+    display: flex;
```
Comment (rittik-ghosh) [inline products.css:21]: Use "beautify" to format the code

File: products.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+.nav-footer {
+    align-items: center;
+    width: 20%;
+}
+
+.btn-nav {
+    
+    margin: 0px;
+    background-color: lightgray;
```
Comment (rittik-ghosh) [inline products.css:38]: there is extra space and you can use variables here too.
Also you can specify margin :0 at the root so you won't have to mention it everywhere

File: products.css [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+.nav-main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 250%;
+
```
Comment (rittik-ghosh) [inline products.css:46]: Same here utility class and extra spacing

File: products.html [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+                    <div class="card-main">
+                       <p class="prod-name">Nike Air Jordans 1</p>
+                       <p class="prod-type">Men Shoes Sneakers</p>
+                       <p class="prod-price">Rs. 30,000</p>
+                    </div>
+                    <button class="btn txt-style">BUY NOW</button>
+                    <button class="btn  "><i class="fas fa-shopping-cart fa-2x"></i></button>
+              
+                 </div> -->
+                 </div>
```
Comment (rittik-ghosh) [inline products.html]: you can remove the commented code

File: products.css [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+.main-container {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.filter-container {position: fixed;
+    
+    left : 1vw;
```
Comment (rittik-ghosh) [inline products.css:68]: there is extra spaces

File: products.css [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+.product-wrapper{position: relative;
+    padding: 1rem;
+    background-color:lightgray;
+    left : 4vw;
+    top : 10vh;
+    display : grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap : 15px;
```
Comment (rittik-ghosh) [inline products.css:127]: you can use rem here because the gap won't change w.r.t the change in size

**Review by rittik-ghosh — COMMENTED** [review]
Have suggested a few changes like:
1) You can use utility classes and variable names to make the code clean
2) You can remove the extra spaces and indent the file using "beautify" for better readability.
3) You can same units to be consistent like gap:1rem.

In interest of time you can refactor your code later

**Comment by TJain02** [comment]
You can remove the comments. Rest looks good.

**Pull request merged**